File: PYTHON/Clase8-CuentaBancaria/Practica4.py
class CuentaBancaria:
    def __init__(self, titular, saldo_inicial):
        self._titular = titular
        self._saldo = saldo_inicial

    def get_titular(self):
        return self._titular

    def get_saldo(self):
        return self._saldo

    def depositar(self, monto):
        if monto > 0:
            self._saldo += monto

    def retirar(self, monto):
        if monto > 0 and monto <= self._saldo:
            self._saldo -= monto

    def mostrar(self):
        print("Titular =", self._titular)
        print("Saldo =", self._saldo)


c1 = CuentaBancaria("Maria", 100000.0)
c1.mostrar()

c1.depositar(25000.0)
c1.retirar(10000.0)

print("\nDespues de operaciones:")
c1.mostrar()

File: C++/Clase8-CuentaBancaria/Practica4.cpp
#include <iostream>
#include <string>
using namespace std;

class CuentaBancaria
{
private:
    string titular;
    double saldo;

public:
    CuentaBancaria(string titular, double saldoInicial)
    {
        this->titular = titular;
        this->saldo = saldoInicial;
    }

    string getTitular()
    {
        return titular;
    }

    double getSaldo()
    {
        return saldo;
    }

    void depositar(double monto)
    {
        if (monto > 0)
            saldo += monto;
    }

    void retirar(double monto)
    {
        if (monto > 0 && monto <= saldo)
            saldo -= monto;
    }

    void mostrar()
    {
        cout << "Titular = " << titular << "\n";
        cout << "Saldo = " << saldo << "\n";
    }
};

int main()
{
    CuentaBancaria c1("Maria", 100000.0);
    c1.mostrar();

    c1.depositar(25000.0);
    c1.retirar(10000.0);

    cout << "\nDespues de operaciones:\n";
    c1.mostrar();

    return 0;
}